docs/data-ai/train/create-dataset.md

```diff
@@ -0,0 +1,153 @@
+
+To create a dataset, pass a unique dataset name and organization ID to [`data_client.create_dataset`](/dev/reference/apis/data-client/#createdataset):
+
+```python
+viam_client = await connect()
+data_client = viam_client.data_client
+
+print("Creating dataset...")
+
+try:
+    dataset_id = await data_client.create_dataset(
+        name="<dataset_name>",
+        organization_id="<org_id>",
+    )
+    print(f"Created dataset: {dataset_id}")
+except Exception as e:
+    print("Error creating dataset. It may already exist.")
+    print(f"Exception: {e}")
+    raise
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+To create a dataset, pass a unique dataset name and organization ID to [`DataClient.CreateDataset`](/dev/reference/apis/data-client/#createdataset):
+
+```go
+ctx := context.Background()
+viamClient, err := client.New(ctx, "<machine_address>", logger)
+if err != nil {
+    log.Fatal(err)
+}
+defer viamClient.Close(ctx)
+
+dataClient := viamClient.DataClient()
+
+fmt.Println("Creating dataset...")
+
+datasetID, err := dataClient.CreateDataset(ctx, &datamanager.CreateDatasetRequest{
+    Name:           "<dataset_name>",
+    OrganizationID: "<org_id>",
+})
+
+if err != nil {
+    fmt.Println("Error creating dataset. It may already exist.")
+    fmt.Printf("Exception: %v\n", err)
+    return
+}
+
+fmt.Printf("Created dataset: %s\n", datasetID)
+```
+
+{{% /tab %}}
+{{% tab name="TypeScript" %}}
+
+To create a dataset, pass a unique dataset name and organization ID to [`dataClient.createDataset`](/dev/reference/apis/data-client/#createdataset):
+
+```typescript
+const client = await createViamClient();
+const dataClient = client.dataClient;
+
+console.log("Creating dataset...");
+
+try {
+  const datasetId = await dataClient.createDataset({
+    name: "<dataset_name>",
+    organizationId: "<org_id>",
+  });
+  console.log(`Created dataset: ${datasetId}`);
+} catch (error) {
+  console.log("Error creating dataset. It may already exist.");
+  console.log(`Exception: ${error}`);
+  process.exit(1);
+}
+```
+
+{{% /tab %}}
+{{% tab name="Flutter" %}}
+
+To create a dataset, pass a unique dataset name and organization ID to [`dataClient.createDataset`](/dev/reference/apis/data-client/#createdataset):
+
+```dart
+final viamClient = await ViamClient.connect();
+final dataClient = viamClient.dataClient;
+
+print("Creating dataset...");
+
+try {
+    final datasetId = await dataClient.createDataset(
+        name: "<dataset_name>",
+        organizationId: "<org_id>",
+    );
+    print("Created dataset: $datasetId");
+} catch (e) {
+    print("Error creating dataset. It may already exist.");
+    print("Exception: $e");
+    return;
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+Once you create a dataset, [capture](/data-ai/train/capture-images/) images, [add](/data-ai/train/update-dataset/) the images to your dataset, and [annotate](/data-ai/train/annotate-images/) the images with training metadata to train your own ML model.
```

Comment:
```suggestion
Once you create a dataset, [capture](/data-ai/train/capture-images/) images, [add the images](/data-ai/train/update-dataset/) to your dataset, and [annotate](/data-ai/train/annotate-images/) the images with training metadata to train your own ML model.
```
[opt] Would be better for accessibility to have the full "capture images" etc. for each of the three items in the link, for more descriptive link text. But I think you did it this way so that it's visually clear that there are 3 different links here? But, "add" is so tiny that maybe expand that one at least? easier to click on haha. Or better yet just delete these links since you have them below and this is redundant.